clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -297,7 +297,8 @@ public long getDefaultBlockSize() {
       ClientBlockInfo info = blocks.get(k);
       long offset = info.getOffset();
       long end = offset + info.getLength();
-      if ((offset >= start && offset <= start + len) || (end >= start && end <= start + len)) {
+      if ((offset >= start && offset <= start + len) || (end >= start && end <= start + len)
```

Comment:
Yes, it is equivalent. I'll change it to `if (end >= start && offset <= start + len)`